schemas/primary_diagnosis.json

```diff
@@ -167,6 +167,7 @@
         "codeList": [
           "AJCC 8th edition",
           "AJCC 7th edition",
+          "AJCC 6th edition",
```

Comment:
Should we jsut make this a standardized codelist in https://raw.githubusercontent.com/icgc-argo/argo-dictionary/master/references/list.json ?

Reply:
@rosibaj Good point. I moved the codelist to list.json (latest commit) and tested the changes in QA using dictionary version `112.3`.